browser-extension/common/src/types.ts

```diff
@@ -43,15 +43,26 @@ export enum SourceOperator {
 }
 
 export interface UrlSource {
+  id?: string;
+  isActive?: boolean;
```

Comment:
Lets create a new interface `PageSource`

Reply:
I think there should not be a separate interface